functions/helpers.bash

```diff
@@ -13,21 +13,22 @@ COL_MAGENTA=$ESC"35;01m"
 COL_CYAN=$ESC"36;01m"
 COL_LGRAY=$ESC"37;01m"
 COL_DGRAY=$ESC"90;01m"
+export COL_DEF COL_RED COL_GREEN COL_YELLOW COL_BLUE COL_MAGENTA COL_CYAN COL_LGRAY COL_DGRAY
```

Comment:
I would say we don't need to export these variables.

Reply:
agree, then again deleting the export line results in 
SC2034: COL_BLUE appears unused. Verify use (or export if used externally).
So it was one line to fix this using export or we would need one shellcheck exception line per color so I chose export. Don't want to disable that SC globally on this.